assets/model_monitoring/components/src/model_monitor_output_metrics/entities/signals/data_drift_signal.py

```diff
@@ -37,7 +37,7 @@ def __init__(
         self.feature_metrics: List[FeatureMetrics] = self._build_feature_metrics(
             monitor_name, signal_name, metrics, feature_importance
         )
-        self.histogram_builder = HistogramBuilder(baseline_histogram, target_histogram)
+        self.histogram_builder = HistogramBuilder(target_histogram, baseline_histogram)
```

Comment:
we have such a bug in the code for quite a long time? So the datadrift we build before are all wrong?

Reply:
yes.